guides/common/modules/proc_integrating-a-local-self-managed-dns-service.adoc

```diff
@@ -0,0 +1,27 @@
+[id="integrating-a-local-self-managed-dns-service"]
+= Integrating a local, self-managed DNS service
```

Comment:
unsure about the comma in the heading, AFAIK this would be a first.

Reply:
I agree it looks a bit off. Would it work without the comma @mmuehlfeldRH?